lighthouse-core/report/html/renderer/crc-details-renderer.js

```diff
@@ -84,9 +84,10 @@ class CriticalRequestChainRenderer {
    * @param {DOM} dom
    * @param {DocumentFragment} tmpl
    * @param {CRCSegment} segment
+   * @param {DetailsRenderer} detailsRenderer
    * @return {Node}
    */
-  static createChainNode(dom, tmpl, segment) {
+  static createChainNode(dom, tmpl, segment, detailsRenderer) {
```

Comment:
this definitely feels weird and passing `this` to anything smells, but I don't really have a better suggestion that doesn't involve totally refactoring how details renderer is structured, sooooooo. sure, why not :)

Reply:
maybe this class should extend DR? but i agree on deferring, this shouldnt be handled within this pr